pkg/ddevapp/global_dotddev_assets/commands/host/sequelace

```diff
@@ -10,7 +10,7 @@
 
 DATABASE="${1:-db}"
 
-query="mysql://root:root@127.0.0.1:${DDEV_HOST_DB_PORT}/${DATABASE}"
+query="mysql://root:root@$${DDEV_PROJECT}.${DDEV_TLD}:${DDEV_HOST_DB_PORT}/${DATABASE}"
```

Comment:
There is an extra `$`.

P.S. I've noticed that in some IDEs you select a Bash variable, it doesn't select the `$`, and when you paste something it doesn't remove the second `$`.